### ISSUE TITLE
How to check empty string or null value?

### ISSUE THREAD

**Issue by chiliec** [editor-js/editorjs-php#43]
For example, Image tool have a caption and it can be **null** or **''**. How to skip such values?
My config:
```php
"image": {
    ...
    "caption": {
        "type": "string",
        "required": false
    },
    ...
}
```
Now returns `Option 'caption' with value `` must be string`

**Comment by khaydarov** [comment]
hi, @chiliec. I'll check it out soon

**Comment by n0str** [comment]
Included experimental solution to the branch: `choose-allowed-null`, introducing optional `allow_null`
```
"string_test": {
  "type": "string",
  "required": false,
  "allow_null": true
}
```

**Comment by khaydarov** [comment]
@chiliec

**Comment by chiliec** [comment]
Works great, thank you so much! 👏

But do you think it always should go together with **required false**? What if value must be in response, but **null** is a valid value?

**Comment by neSpecc** [comment]
Resolved by #44
